README.md

```diff
@@ -77,12 +88,12 @@ line option. To disable this functionality altogether, use
 
 ## Build
 
-To be able to build from source code, you need to have a working Go environment with [verion 1.7 or greater](https://golang.org/doc/install) installed.
+To be able to build from source code, you need to have a working Go environment with [version 1.7 or greater](https://golang.org/doc/install) installed.
 
 Follow the steps here to build using `make`:
 
-    $ mkdir -p $GOPATH/src/tmobile/collectd_exporter
-    $ cd $GOPATH/src/tmobile/collectd_exporter
+    $ mkdir -p $GOPATH/src/tmobile
+    $ cd $GOPATH/src/tmobile
     $ git clone git@github.com:dev9com/collectd_exporter.git
```

Comment:
need to point to bitbucket  ssh://git@bitbucket.corporate.t-mobile.com/ucm/collectd_exporter.git